spec/models/event_stream_spec.rb

```diff
@@ -1,27 +1,4 @@
 RSpec.describe EventStream do
-  describe ".event_groups" do
-    EventStream.event_groups.each do |group_name, group_data|
-      (EmsEvent.group_levels + MiqEvent.group_levels).each do |level|
```

Comment:
The old code checked MiqEvent as well (not sure why) - should this test also be in the MiqEvent subclass?

Reply:
It could but most if not all [MiqEvents](https://github.com/ManageIQ/manageiq/blob/master/db/fixtures/miq_event_definition_events.yml) don't specify a level/severity so they all show up as `:detail` in timelines as that's the only thing that made sense.  When we specify the correct levels for our events, we can add the tests then.

Reply:
It was a bug to add MiqEvent in the first place so I removed it.